games-emulation/ryujinx/ryujinx-1.1.1221.ebuild

```diff
@@ -0,0 +1,404 @@
+system.reflection.primitives@4.0.1
+system.reflection.primitives@4.3.0
+system.reflection.typeextensions@4.1.0
+system.reflection.typeextensions@4.3.0
+system.resources.resourcemanager@4.0.1
+system.resources.resourcemanager@4.3.0
+system.runtime@4.1.0
+system.runtime@4.3.0
+system.runtime.compilerservices.unsafe@4.7.1
+system.runtime.compilerservices.unsafe@5.0.0
+system.runtime.compilerservices.unsafe@6.0.0
+system.runtime.extensions@4.1.0
+system.runtime.extensions@4.3.0
+system.runtime.handles@4.0.1
+system.runtime.handles@4.3.0
+system.runtime.interopservices@4.1.0
+system.runtime.interopservices@4.3.0
+system.runtime.interopservices.runtimeinformation@4.0.0
+system.runtime.numerics@4.0.1
+system.security.accesscontrol@4.5.0
+system.security.claims@4.3.0
+system.security.cryptography.algorithms@4.2.0
+system.security.cryptography.cng@4.2.0
+system.security.cryptography.csp@4.0.0
+system.security.cryptography.encoding@4.0.0
+system.security.cryptography.openssl@4.0.0
+system.security.cryptography.primitives@4.0.0
+system.security.cryptography.x509certificates@4.1.0
+system.security.principal@4.3.0
+system.security.principal.windows@4.3.0
+system.security.principal.windows@4.5.0
+system.text.encoding@4.0.11
+system.text.encoding@4.3.0
+system.text.encoding.codepages@4.5.1
+system.text.encoding.codepages@5.0.0
+system.text.encoding.codepages@7.0.0
+system.text.encoding.extensions@4.0.11
+system.text.encodings.web@6.0.0
+system.text.json@6.0.0
+system.text.regularexpressions@4.1.0
+system.threading@4.0.11
+system.threading@4.3.0
+system.threading.overlapped@4.3.0
+system.threading.tasks@4.0.11
+system.threading.tasks@4.3.0
+system.threading.tasks.extensions@4.0.0
+system.threading.tasks.extensions@4.5.4
+system.threading.threadpool@4.3.0
+system.threading.timer@4.0.1
+system.xml.readerwriter@4.0.11
+system.xml.xdocument@4.0.11
+tmds.dbus.protocol@0.15.0
+unicornengine.unicorn@2.0.2-rc1-fb78016
+"
+
+inherit check-reqs desktop dotnet-pkg xdg
+
+DESCRIPTION="Experimental Nintendo Switch Emulator written in C#"
+HOMEPAGE="https://www.ryujinx.org/
+	https://github.com/Ryujinx/Ryujinx/"
+
+if [[ "${PV}" == *9999* ]] ; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/${PN^}/${PN^}.git"
+else
+	SRC_URI="https://github.com/${PN^}/${PN^}/archive/${PV}.tar.gz
+		-> ${P}.tar.gz"
+	S="${WORKDIR}/${PN^}-${PV}"
+
+	KEYWORDS="~amd64"
+fi
+
+SRC_URI+=" ${NUGET_URIS} "
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="
+	app-arch/brotli
+	dev-libs/expat
+	dev-libs/icu
+	dev-libs/libxml2
+	dev-libs/openssl
+	dev-libs/wayland
+	media-gfx/graphite2
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/harfbuzz
+	media-libs/libglvnd[X]
+	media-libs/libpng
+	media-libs/libpulse
+	media-libs/libsdl2
+	media-video/pipewire
+	x11-libs/gtk+:3
+	x11-libs/libX11
+"
+
+CHECKREQS_DISK_BUILD="3G"
+DOTNET_PKG_PROJECTS=(
+	"src/${PN^}/${PN^}.csproj"
+)
+PATCHES=(
+	"${FILESDIR}/${P}-better-defaults.patch"
+	"${FILESDIR}/${P}-disable-updates.patch"
```

Comment:
please leave the version (PV) hardcoded, easier to reuse patches if they are needed